### ISSUE TITLE
Create better JavaDoc

### ISSUE THREAD

**Issue by aslakhellesoy** [cucumber-attic/cuke4duke#41]
There is nothing!

**Comment by gaffo** [comment]
+1

**Comment by martinsson** [comment]
Id be quite satisfied if the sources were available in the maven repo. If I'd fork and add this to the pom, would you merge into master? :
        <plugin>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-source-plugin</artifactId>
            <executions>
                <execution>
                    <id>bind-sources</id>
                    <goals>
                        <goal>jar</goal>
                    </goals>
                </execution>
            </executions>
        </plugin>

**Comment by aslakhellesoy** [comment]
Go for it mate!

**Comment by aslakhellesoy** [comment]
Does this mean that a mvn deploy will upload the sources in a jar file next to the binary jars?

**Comment by martinsson** [comment]
exactly, you'll have : cuke4duke-0.2.5-sources.jar. Then when I click on table.mapToColumn() to see what it does my IDE downloads the sources and displays them automatically, making my life dead simple :)

**Comment by aslakhellesoy** [comment]
So - when I get round to writing Javadoc for the APIs - would you then get JavaDoc in the IDE too? (assuming you don't want to read source code). -Or would I have to upload a Javadoc jar file too? Ideally I want to spare folks from reading source code to learn the API, but I'm not familiar enough with what's required fior that...

**Comment by martinsson** [comment]
Yes you get the javadoc with the sources. Or at least it works in eclipse with the m2 plugin. 
I downloaded cuke4duke, made the modification to the root pom, 
mvn clean install
And sources were installed to my local repo.

I just sent you a pull request.

Cheers and thanks for your great work.
Johan
